appserver/transaction/jts/src/main/java/com/sun/jts/CosTransactions/SubCoordinator.java

```diff
@@ -1738,7 +1736,7 @@ public void getResources(ResourceSequenceHolder resources,
      *
      * @see
      */
-    CompletionHandler getTerminator() {
+    synchronized CompletionHandler getTerminator() {
```

Comment:
this could change behaviour what is the rationale behind this change?

Reply:
To mirror the synchronization from the `setTerminator` Method. Please see https://www.securecoding.cert.org/confluence/x/I4BoAg